src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java

```diff
@@ -388,6 +390,47 @@ public void whenStatsOnNotInitializedSubscriptionThanCorrectResponse() throws IO
         }
     }
 
+    @Test
+    public void whenStreamDuplicatePartitionsThenUnprocessableEntity() throws IOException {
+        final String et = createEventType().getName();
+        final Subscription s = createSubscriptionForEventType(et);
+
+        final String body = "{\"partitions\":[" +
+                "{\"event_type\":\"et1\",\"partition\":\"0\"}," +
```

Comment:
Shouldn't you use `et` instead of `"et1"`? (same next line)

Reply:
I think I didn't get the question.
This test tests that Nakadi checks if user specified duplicated partition, so I need to provide the same event-type:partition pair twice as an input.